pycbc/inference/prior.py

```diff
@@ -28,11 +28,168 @@
 
 import numpy
 import scipy.stats
+from ConfigParser import Error
 
 #
 #   Distributions for priors
 #
-class Uniform(object):
+
+# helper function for getting bounds from config file
+def get_param_bounds_fromcp(cp, section, tag, param):
+    """
+    Gets bounds for the given parameter from a section in a config file.
+    Bounds are specified by setting the parameter name equal to a comma-
+    separated minimum and maximum, e.g., `foo = 3.2,4.1`. If the parameter is
+    not found in the section, or the parameter is not equal to anything (i.e.,
+    `foo =`), will just return None. If bounds are specified, both a minimum
+    and maximum must be provided, else a Value or Type Error will be raised.
+
+    Parameters
+    ----------
+    cp : ConfigParser instance
+        The config file.
+    section : str
+        The name of the section.
+    tag : str
+        Any tag in the section name. The full section name searched for in
+        the config file is `{section}(-{tag})`.
+    param : str
+        The name of the parameter to retrieve bounds for.
+
+    Returns
+    -------
+    bounds : {tuple | None}
+        If bounds were provided, a tuple giving the minimum and maximum value,
+        as floats. Otherwise, None.
+    """
+    try:
+        bnds = cp.get_opt_tag(section, param, tag)
+    except Error:
+        bnds = None
+    if bnds is not None:
+        bnds = bnds.split(',')
+        if len(bnds) != 2:
+            raise ValueError("if specifying bounds for %s, " %(param) +
+                "you must provide both a minimum and a maximum")
+        bnds = map(float, bnds)
+    return bnds
+
+
+class BoundedDist(object):
```

Comment:
If this class isn't being included in the ``pycbc.inference.prior.prior`` dict then can we rename it like ``_BoundedDistribution``? It seems completely internal to the module. It's also only going to be used as a superclass so I don't see why we can't be more verbose with the class name.